src/Core/Migration/Traits/Translations.php

```diff
@@ -9,17 +9,13 @@ class Translations
 {
     /**
      * @var array<string, string|null>
-     *
-     * @deprecated tag:v6.7.0 - Will be natively typed
      */
-    protected $german;
+    protected array $german;
```

Comment:
use constructor property promotion, then we alson only need to define the array shape once